src/NServiceBus.Metrics.ServiceControl.Tests/When_reporting_to_ServiceControl_expires.cs

```diff
@@ -0,0 +1,127 @@
+﻿namespace NServiceBus.Metrics.ServiceControl.Tests
+{
+    using System;
+    using System.Threading;
+    using AcceptanceTesting;
+    using Config;
+    using Config.ConfigurationSource;
+    using EndpointTemplates;
+    using NUnit.Framework;
+    using Pipeline;
+    using Pipeline.Contexts;
+
+    public class When_reporting_to_ServiceControl_expires : NServiceBusAcceptanceTest
+    {
+        static readonly TimeSpan TTBR = TimeSpan.FromSeconds(10);
+
+        [Test]
+        public void Should_report_nothing_when_ttbr_breached()
+        {
+            var context = new Context();
+
+            Scenario.Define(context)
+                .WithEndpoint<Sender>(b => b.Given((bus, c) =>
+                {
+                    bus.SendLocal(new MyMessage());
+                }))
+                .Done(ctx => ctx.MessageProcessedBySender)
+                .Run();
+
+            Thread.Sleep(TTBR + TTBR);
+
+            Scenario.Define(context)
+                .WithEndpoint<MonitoringMock>()
+                .Done(ctx => ctx.WasCalled)
+                .Run(TimeSpan.FromSeconds(10));
+
+            Assert.IsFalse(context.WasCalled);
+        }
+
+        [Test]
+        public void Should_report_when_ttbr_not_breached()
+        {
+            var context = new Context();
+
+            Scenario.Define(context)
+                .WithEndpoint<Sender>(b => b.Given((bus, c) =>
+                {
+                    bus.SendLocal(new MyMessage());
+                }))
+                .WithEndpoint<MonitoringMock>()
+                .Done(ctx => ctx.WasCalled)
+                .Run();
+
+            Assert.True(context.WasCalled);
+        }
+
+        public class Context : ScenarioContext
+        {
+            public bool MessageProcessedBySender { get; set; }
+            public bool WasCalled { get; set; }
+        }
+
+        public class Sender : EndpointConfigurationBuilder
+        {
+            public Sender()
+            {
+                EndpointSetup<DefaultServer>(cfg =>
+                {
+                    var monitoringQueue = AcceptanceTesting.Customization.Conventions.EndpointNamingConvention(typeof(MonitoringMock));
+
+                    cfg.SendMetricDataToServiceControl(monitoringQueue, "my-custom-instance");
+                    cfg.SetServiceControlMetricsMessageTTBR(TTBR);
+                    cfg.Transactions().Disable(); //transactional msmq with ttbr not supported
```

Comment:
> transactional msmq with ttbr not supported

Reply:
What did you mean @ramonsmits ?

Reply:
Ow... strange... I thought I had corrected this comment :-)

Reply:
I'm not sure why that explicitly mentions MSMQ TTBR, I think we do not require transactions at all on any transport AFAIK?